lib/preprocess.js

```diff
@@ -41,6 +42,13 @@ export class Measurement {
     this.stationId = pointerize(m.station_id)
     this.carChecksum = pointerize(m.car_checksum)
     this.carTooLarge = m.car_too_large
+    this.networkRetrieval = {
```

Comment:
If we agree to use a different name than "networkRetrieval", then we need to rename this property too.

See https://github.com/CheckerNetwork/spark-checker/pull/132#discussion_r2037288495 and https://github.com/CheckerNetwork/spark-checker/pull/132#discussion_r2037309490

Plus, all other places that use the name "networkRetrieval" 😢